src/init/init.js

```diff
@@ -1,10 +1,18 @@
 const nodePlop = require("node-plop");
-
 const registryUrl = require('registry-url');
+
+var os = require('os');
+var path = require('path');
+
 const PluginManager = require('live-plugin-manager').PluginManager;
-const manager = new PluginManager({npmRegistryUrl: registryUrl()})
+const manager = new PluginManager({
+    npmRegistryUrl: registryUrl(),
+    pluginsPath: path.join(os.tmpdir(), 'plugin_packages')
```

Comment:
to avoid potential naming conflict, we should name it something like `apikana-plugin-packages`